worker/package.json

```diff
@@ -1,8 +1,31 @@
 {
-  "private": true,
   "name": "worker",
-  "version": "0.0.1",
-  "description": "Cloudflare Workers for deno.land",
-  "main": "index.js",
-  "license": "MIT"
+  "version": "1.0.0",
+  "description": "Cloudflare worker for deno.land",
+  "license": "MIT",
+  "scripts": {
+    "build": "webpack",
+    "dev": "NODE_ENV=development npm run build",
+    "format": "prettier --write 'src/**/*.{ts,js,css,json,md}'",
+    "test:clean": "rimraf ./transpiled/shared ./transpiled/worker",
+    "test": "npm run test:clean && npm run transpile && mocha --require source-map-support/register --recursive transpiled/worker/test",
```

Comment:
You use yarn.lock but call `npm run` here. I feel like we should use one or the other but not both.